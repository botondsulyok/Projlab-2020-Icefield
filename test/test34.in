field start 0
field end 0
neighbours start end
neighbours end start
entity pb icebear start
entity esk1 Eskimo end
entity esk2 Eskimo end
ability end esk1
stats start
stats end
stats esk1
stats esk2
stats pb
move pb end
stats start
stats end
stats esk1
stats esk2
stats pb