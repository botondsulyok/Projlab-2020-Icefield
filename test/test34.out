id: start
type: iceunit
building: plains0
snow: 0
pickable: null
standers: pb
pullers: null
masters: null
neighbours: end

id: end
type: iceunit
building: igloo0
snow: 0
pickable: null
standers: esk1, esk2
pullers: esk1, esk2
masters: esk1, esk2
neighbours: start

id: esk1
type: eskimo
iceUnit: end
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk2
type: eskimo
iceUnit: end
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: pb
type: icebear
iceUnit: start
movebehaviour: amph0

Game Over
id: start
type: iceunit
building: plains0
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: end

id: end
type: iceunit
building: igloo0
snow: 0
pickable: null
standers: esk1, esk2, pb
pullers: esk1, esk2
masters: esk1, esk2
neighbours: start

id: esk1
type: eskimo
iceUnit: end
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: esk2
type: eskimo
iceUnit: end
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: pb
type: icebear
iceUnit: end
movebehaviour: amph0

